.github/workflows/ci.yaml

```diff
@@ -33,13 +50,112 @@ jobs:
+      run: exit 1
+    - uses: actions/checkout@v2
     - uses: jpkrohling/setup-operator-sdk@v1.1.0
       with:
         operator-sdk-version: v1.28.0
-    - run: make generate manifests manager test-envtest
-      if: ${{ github.event_name == 'pull_request' }}
-    - run: make oci-build
-      if: ${{ github.event_name == 'push' }}
+    - name: Build scorecard image for test
+      id: build-scorecard
+      run: |
+        make scorecard-build CUSTOM_SCORECARD_IMG=ghcr.io/${{ github.repository_owner }}/cryostat-operator-scorecard:ci-$GITHUB_SHA
+        echo "tag=ci-$GITHUB_SHA" >> $GITHUB_OUTPUT
+    - name: Push scorecard image to ghcr.io for test
+      id: push-scorecard-to-ghcr
+      uses: redhat-actions/push-to-registry@v2
+      with:
+        image: cryostat-operator-scorecard
+        tags: ${{ steps.build-scorecard.outputs.tag }}
+        registry: ghcr.io/${{ github.repository_owner }}
+        username: ${{ github.repository_owner }}
+        password: ${{ secrets.GHCR_PR_TOKEN }}
+    - name: Build bundle image for test
+      id: build-bundle
+      run: |
+        make bundle bundle-build BUNDLE_IMG=ghcr.io/${{ github.repository_owner }}/cryostat-operator-bundle:ci-$GITHUB_SHA
+        echo "tag=ci-$GITHUB_SHA" >> $GITHUB_OUTPUT
+    - name: Push bundle image to ghcr.io for test
+      id: push-bundle-to-ghcr
+      uses: redhat-actions/push-to-registry@v2
+      with:
+        image: cryostat-operator-bundle
+        tags: ${{ steps.build-bundle.outputs.tag }}
+        registry: ghcr.io/${{ github.repository_owner }}
+        username: ${{ github.repository_owner }}
+        password: ${{ secrets.GHCR_PR_TOKEN }}
+    - name: Set up Kind cluster
+      run: |
+        {
+            printf -- "kind: Cluster\n"
+            printf -- "apiVersion: kind.x-k8s.io/v1alpha4\n"
+            printf -- "nodes:\n"
+            printf -- "- role: control-plane\n"
+            printf -- "  kubeadmConfigPatches:\n"
+            printf -- "  - |\n"
+            printf -- "    kind: InitConfiguration\n"
+            printf -- "    nodeRegistration:\n"
+            printf -- "      kubeletExtraArgs:\n"
+            printf -- "        node-labels: \"ingress-ready=true\"\n"
+        } > kind-config.yaml
+        kind create cluster --config=kind-config.yaml -n ci-${{ github.run_id }}
+        # Enabling Ingress
+        kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/main/deploy/static/provider/kind/deploy.yaml
+        kubectl rollout status -w deployment/ingress-nginx-controller -n ingress-nginx --timeout 5m
+    - name: Install Operator Lifecycle Manager
+      run: curl -sL https://github.com/operator-framework/operator-lifecycle-manager/releases/download/v0.24.0/install.sh | bash -s v0.24.0
+    - name: Install Cert Manager
+      run: make cert_manager
+    - uses: docker/login-action@v2
```

Comment:
Does `KIND_EXPERIMENTAL_PROVIDER=podman kind create cluster` work? If not we can stick with Docker.

Reply:
...unfortunately, it does not seem to work:

```
INFO[0000] trying next host                              error="failed to authorize: failed to fetch anonymous token: unexpected status: 401 Unauthorized" host=ghcr.io
FATA[0000] Failed to run bundle: pull bundle image: error pulling image ghcr.io/tthvo/cryostat-operator-bundle:ci-abc78d7db8a6080f9bbe602c01e93e60970f8733: error resolving name for image ref ghcr.io/tthvo/cryostat-operator-bundle:ci-abc78d7db8a6080f9bbe602c01e93e60970f8733: failed to authorize: failed to fetch anonymous token: unexpected status: 401 Unauthorized 
```

Reply:
Was testing with my ghcr private image here: `ghcr.io/tthvo/cryostat-operator-bundle:ci-abc78d7db8a6080f9bbe602c01e93e60970f8733`

Reply:
It seems like the Operator SDK does indeed hardcode ~/docker/config.json. Can you try with the podman-login action and adding the following parameter: `auth_file_path: ${{ env.HOME }}/.docker/config.json`?

Reply:
I am running into this issue on CI:

```
Run redhat-actions/podman-login@v1
/usr/bin/podman version
  /usr/bin/podman version
  Version:      3.4.4
  API Version:  3.4.4
  Go Version:   go1.17.3
  Built:        Thu Jan  1 00:00:00 1970
  OS/Arch:      linux/amd64
/usr/bin/podman login ghcr.io -u tthvo -p *** --verbose --authfile=/.docker/config.json
Error: 1 error occurred:
	* mkdir /.docker: permission denied


Error: Error: podman exited with code 125
Error: 1 error occurred:
	* mkdir /.docker: permission denied
```

Reply:
`env.HOME` is undefined/empty, I guess?

https://docs.github.com/en/actions/learn-github-actions/variables

Reply:
`RUNNER_TEMP` might be useful?

Reply:
Oh wow right! Lets try that!

Reply:
Weird.

![image](https://github.com/cryostatio/cryostat-operator/assets/3787464/fe687e3b-33bb-4ba9-ba2e-c6b11a5183a9)

Reply:
Maybe the `${{ env.FOO }}` syntax is only for env vars that are defined on the org/repo?

![image](https://github.com/cryostatio/cryostat-operator/assets/3787464/4ca26bf1-7594-4524-ad3b-e3742915bc22)

Perhaps just try using `$RUNNER_TEMP` instead of `${{ env.RUNNER_TEMP }}`

Reply:
I guess that's what this section is talking about, but I haven't really read it:

https://docs.github.com/en/actions/learn-github-actions/variables#using-the-env-context-to-access-environment-variable-values

Reply:
Or maybe just `$HOME`?

Reply:
Yess `$HOME` worked!

Reply:
Awesome! Thanks for sticking with this. I prefer using Podman/Buildah tooling where possible.